stl/inc/vector

```diff
@@ -3041,7 +3041,7 @@ template <class _Alloc>
 _INLINE_VAR constexpr bool _Is_vb_iterator<_Vb_const_iterator<_Alloc>, false> = true;
 
 template <class _VbIt, class _Ty>
-_CONSTEXPR20 void _Fill_vbool(_VbIt _First, _VbIt _Last, const _Ty& _Val) {
+_CONSTEXPR20 void _Fill_vbool(_VbIt _First, _VbIt _Last, const bool _Val) noexcept {

```

Comment:
So the issue in CI is that it tries to pass `const bool&` but finds `const bool`. I think we can generalize here and create a temporary `const bool _BVal = _Val;` and pass that to `_Find_vbool`.

I am 99% sure that we could drop the `is_same_v<_Ty, bool>` in that case, although we would accept types that are implicitly convertible to bool like `find(first, last, MyPtr);`

Reply:
Is accepting implicitly convertible to `bool` bad or good? What does the Standard want?

Reply:
`find` and `count` have this check for exact `bool`, whereas `fill` does not

Reply:
So the issue really was that I forgot to remove the previously-deduced template parameter